sirepo/package_data/static/json/schema-common.json

```diff
@@ -131,15 +127,13 @@
         "deleteSimulation": "/delete-simulation",
         "downloadDataFile": "/download-data-file/<simulation_type>/<simulation_id>/<model>/<frame>/?<suffix>",
         "downloadFile": "/download-file/<simulation_type>/<simulation_id>/<filename>",
-        "errorLogging": "/error-logging",
```

Comment:
This is a bit odd. I get that we never look up this value so we don't need it here. But, now we have routes that aren't listed in the schema but that exist which seems odd. 

While looking at this code I came across this comment which I don't think makes any sense https://github.com/radiasoft/sirepo/blob/423a03e8c432a9a9a2cca8adc589955e9927d740/sirepo/package_data/static/js/sirepo.js#L3310

Reply:
e4183ec27